src/browser/modules/Stream/FrameTitlebar.jsx

```diff
@@ -81,28 +83,54 @@ class FrameTitlebar extends Component {
     })
     saveAs(blob, 'export.csv')
   }
+
+  exportTXT = () => {
+    const { frame } = this.props
+
+    if (frame.type === 'history') {
+      const asTxt = frame.result.join(';\n\n')
```

Comment:
Could you check so the statement isn't already ending with a semicolon?
Also, client commands (statements starting with `:`) should / cannot end with a semicolon, only Cypher.

Reply:
done!